Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: clean compile_translations dummy_translations \
+	extract_translations fake_translations help
+
+.DEFAULT_GOAL := help
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## remove generated byte code, coverage reports, and build artifacts
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
+
+## Localization targets
+
+extract_translations: ## extract strings to be translated, outputting .po files
+	cd drag_and_drop_v2 && DJANGO_SETTINGS_MODULE="drag_and_drop_v2.locale.settings" django-admin makemessages -l en -v1
+	cd drag_and_drop_v2 && DJANGO_SETTINGS_MODULE="drag_and_drop_v2.locale.settings" django-admin makemessages -l en -v1 -d djangojs --ignore public/js/vendor --ignore public/js/translations
```

Comment:
DRY. Can we store `public/js/translations` in a separate variable as it's used multiple times in the file?